### PR TITLE
Add Support to PHP 8.2

### PR DESCRIPTION
**Pull request by EHER**

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -39,7 +39,7 @@
         }
     },
     "require": {
-        "php": "~8.1.0",
+        "php": "^8.1 || ^8.2",
```
Comment (malukenho) [inline composer.json]: `~8.1.0 || ~8.2.0`

**Review by malukenho — DISMISSED** [review]

**Comment by malukenho** [comment]
Btw, qodana is so nice: https://github.com/codelicia/xulieta/actions/runs/3653544845/jobs/6173126746#step:10:294

File: composer.json (hunk 1 above)
Comment (EHER) [reply composer.json]: In this case, using [tilde](https://getcomposer.org/doc/articles/versions.md#tilde-version-range-) or [caret](https://getcomposer.org/doc/articles/versions.md#caret-version-range-) would be the same, right?

If I understood correctly both use [semantic version](https://semver.org/) but the tilde will always allow an increment of the last digit (except major) while caret always allows patch.

Comment (EHER) [reply composer.json]: Let's align it on Slack so we can have a standard for Codelicia 😄 

For this one I am moving with your suggestion.

**Pull request merged**